client/post-editor/media-modal/index.jsx

```diff
@@ -473,6 +475,7 @@ export class EditorMediaModal extends Component {
 		}
 
 		const selectedItems = this.props.mediaLibrarySelectedItems;
+		const galleryViewEnabled = this.props.galleryViewEnabled;
```

Comment:
While we are here we could do a bit of cleanup here with some object destructuring on this.props for `galleryViewEnabled` and various other items used in props in this method.

Reply:
Sure- good idea!  I cleaned up the props in this function a bit.